client/blocks/data-center-picker/index.tsx

```diff
@@ -23,297 +16,148 @@ interface ExternalProps {
 type Props = ExternalProps & LocalizeProps;
 
 const DataCenterOptions = [
+	{
+		value: '',
+		get label(): string {
+			return translate( 'Optimal data center' );
+		},
+	},
 	{
 		value: 'bur',
-		name: 'geo_affinity',
 		get label(): string {
 			return translate( 'US West' );
 		},
-		thumbnail: {
-			imageUrl: burImg,
-		},
 	},
 	{
 		value: 'dfw',
-		name: 'geo_affinity',
 		get label(): string {
 			return translate( 'US Central' );
 		},
-		thumbnail: {
-			imageUrl: dfwImg,
-		},
 	},
 	{
 		value: 'dca',
-		name: 'geo_affinity',
 		get label(): string {
 			return translate( 'US East' );
 		},
-		thumbnail: {
-			imageUrl: dcaImg,
-		},
 	},
 	{
 		value: 'ams',
-		name: 'geo_affinity',
 		get label(): string {
 			return translate( 'EU West' );
 		},
-		thumbnail: {
-			imageUrl: amsImg,
-		},
 	},
 ];
 
 const Form = styled.div( {
 	maxWidth: '564px',
-} );
-
-const FormHeadingContainer = styled.div( {
-	display: 'flex',
-	alignItems: 'center',
-	marginBottom: '8px',
-} );
-
-const FormHeading = styled.h3( {
-	fontWeight: 600,
-	marginRight: '8px',
-} );
-
-const FormDescription = styled.p( {
-	marginBottom: '12px',
-} );
-
-const AutomaticFormLabel = styled.label( {
-	display: 'block',
-	marginBottom: '12px',
-	width: '100%',
-	maxWidth: '541px',
-	cursor: 'pointer',
-} );
-
-const CompactLabel = styled.label( {
-	cursor: 'pointer',
-	display: 'flex',
-	padding: '24px 16px',
-	border: '2px solid #DCDCDE',
+	border: '1px solid var( --color-border-subtle )',
 	borderRadius: '4px',
-
-	'&:hover': {
-		background: '#F9FBFD',
-	},
-
-	'&:has(input[type=radio]:checked)': {
-		background: '#F9FBFD',
-		borderColor: '#0675C4',
-	},
+	padding: '24px',
 } );
 
-const PickerOptions = styled.div( { display: 'flex', flexDirection: 'column', gap: '16px' } );
-
-const PickerOption = styled.div( {
-	display: 'flex',
-	flexDirection: 'column',
-	justifyContent: 'flex-start',
-	alignItems: 'flex-start',
-	gap: '8px',
+const FormHidden = styled.div( {
+	fontSize: '0.75rem',
 } );
 
-const PickerOptionLabel = styled.span( { lineHeight: '20px', fontWeight: 600, color: '#101517' } );
-
-const PickerRadio = styled.input( {
-	margin: '2px 16px 0 0',
-} );
+const CustomizeLink = styled.a`
+	text-decoration: underline;
+	cursor: pointer;
+`;
 
-const CompactPickerSelect = styled( FormSelect )( {
-	width: 'fit-content',
-	borderRadius: '4px',
-} );
+const SupportLink = styled.a`
+	text-decoration: underline;
+	color: var( --color-text-subtle ) !important;
```

Comment:
Not a big fan of using `!important`. Is this style key to the design or UX? If so, that's fine. I just want to ask.

Reply:
It was to match the design, but I'm not a fan either. I removed those styles.